### ISSUE TITLE
Loss Function Question

### ISSUE THREAD

**Issue by tom-bu** [princeton-vl/CER-MVS#6]
I was looking at loss.py. What is the goal of line 27: `flow_loss += .01 * i_weight * (i_loss).mean()`. If you include this term, shouldn't it include the `valid` mask so that it's `flow_loss += .01 * i_weight * (valid * i_loss).mean()`?

**Comment by mazeyu** [comment]
the intuition behind this is to encourage the model to output 0 disparity when there is no gt, e.g., in the background, at least it helps visually. I am not sure how this could affect quantitative evaluation, but hopefully, it should reduce noisy points.
